src/doc/guide.md

```diff
@@ -408,6 +411,65 @@ will not fail your overall build. Please see the [Travis CI Rust
 documentation](https://docs.travis-ci.com/user/languages/rust/) for more
 information.
 
+# Templates
```

Comment:
This documentation doesn't seem to mention `--template name` along with `~/.cargo/templates`, but maybe that won't be used that much in practice?